stacks/kubesphere/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add test https://charts.kubesphere.io/test
```

Comment:
To confirm, is the name of the helm repo test in production as well?

Reply:
> The files are currently not executable, please changes the file permissions.
Execute permissions have been added

Reply:
Yes, but this repo name and  URL may mislead the novices. We will replace this URL with `main` and `https://charts.kubesphere.io/test`.

Reply:
This has been changed to the main library  @keladhruv  @FeynmanZhou